dashboard/v1.1/src/pages/Input/Input.tsx

```diff
@@ -104,6 +143,33 @@ const Input = (props: InputScreenProps) => {
     }
   }, [body, headers, method, params, route, screenType, labels]);
 
+  useEffect(() => {
+    // Fetch existing routes
+    (async () => {
+      let response;
+
+      try {
+        response = await fetch(`${HOST_IP}/routes-summary`);
+      } catch (err) {
+        console.log(err);
+      }
+
+      const { data } = await response.json();
+      const routes = data.monitoringRoutes;
+      const existingRoutes: string[] = [];
```

Comment:
Can you please use some other constant name as `existingRoutes` have already been declared in line 125?

Reply:
Sure!